src/index.js

```diff
@@ -21,6 +21,7 @@ async function gameFlow () {
     await player.setupBoard();
```

Comment:
This also should not need a review these changes are all related to me simplifying game flow for new computer